_posts/0000-01-02-Buzz-your-girlfriend-woof.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+go to your room: "Welcome to our second slide!"
+---
+drink soda, add file, pee bed
```

Comment:
![image](https://user-images.githubusercontent.com/75340837/100818668-99da1d80-3418-11eb-8dd9-e6edccdd1f1f.png)

Reply:
look what you did you little jerk